templates/conversationQuickReply.tpl

```diff
@@ -2,10 +2,13 @@
 	<article class="message messageSidebarOrientation{$__wcf->getStyleHandler()->getStyle()->getVariable('messageSidebarOrientation')|ucfirst}{if $__wcf->getUserProfileHandler()->userOnlineGroupID} userOnlineGroupMarking{$__wcf->getUserProfileHandler()->userOnlineGroupID}{/if}">
 		{include file='messageSidebar' userProfile=$__wcf->getUserProfileHandler()->getUserProfile() isReply=true enableMicrodata=false}
 		
-		<div class="messageContent messageQuickReplyContent"{if $pageNo < $pages} data-placeholder="{lang}wcf.conversation.reply{/lang}"{/if}>
+		<div class="messageContent messageQuickReplyContent"{if $pageNo <p $pages} data-placeholder="{lang}wcf.conversation.reply{/lang}"{/if}>
```

Comment:
```suggestion
		<div class="messageContent messageQuickReplyContent"{if $pageNo < $pages} data-placeholder="{lang}wcf.conversation.reply{/lang}"{/if}>
```

Reply:
whoops.

Reply:
To have it noted, this compiled just fine and resulted in:

```
<?php if ($this->v['pageNo'] < 'p$pages') { ?>
```
:facepalm: